.github/workflows/deployment-ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
   build:
     if: |
       !contains(github.event.head_commit.message, '[ci skip]') && !contains(github.event.pull_request.title, '[ci skip]')
-      && !(github.repository == 'kordlib/kord' && github.event_name == 'pull_request')
+      && (github.repository != 'kordlib/kord' || github.event_name != 'pull_request')
```

Comment:
The change to the condition should actually make no difference, see https://en.wikipedia.org/wiki/De_Morgan%27s_laws (`!(a && b) == (!a || !b)`).
I think the actual problem is that `github.repository` always is `'kordlib/kord'`, we need the repo where the PR comes from.